src/app/cube/content-pages/about-us/about-us.component.html

```diff
@@ -20,10 +20,12 @@ <h3>{{sec.title}}</h3>
             <div class="content">
               <div [innerHTML]="questions.answer"></div>
               <div *ngIf="questions.video">
-                <iframe src="https://www.youtube.com/embed/ymerRpfbnQQ" title="Clark Searching and Sorting Video"
+                <iframe title="Clark Searching and Sorting Video"
                   frameborder="0"
                   allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-                  allowfullscreen>
+                  allowfullscreen
+                  src="https://www.youtube.com/embed/ymerRpfbnQQ"
```

Comment:
Was this angular?

Reply:
This might have been linting. This page hasn't been touched in a while

Reply:
This was me. I was following the recommendation here: https://angular.io/errors/NG0910